src/js/utils/log.js

```diff
@@ -22,17 +25,19 @@ let log;
  *         but this is exposed as a parameter to facilitate testing.
  */
 export const logByType = (type, args, stringify = !!IE_VERSION && IE_VERSION < 11) => {
+  const lvl = log.levels[level];
+  const lvlRegExp = new RegExp(`^(${lvl})$`);
 
   if (type !== 'log') {
 
-    // add the type to the front of the message when it's not "log"
+    // Add the type to the front of the message when it's not "log".
     args.unshift(type.toUpperCase() + ':');
   }
 
-  // add to history
-  log.history.push(args);
+  // Add a clone of the args at this point to history.
+  log.history.push([].concat(args));
```

Comment:
Changed this to a clone because I think the intent was for `VIDEOJS:` to _not_ get added to history, but it didn't work that way.